src/tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -493,6 +493,20 @@ public ApkBuilder(TaskLoggingHelper logger)
             // NOTE: we can run android-strip tool from NDK to shrink native binaries here even more.
 
             File.Copy(dynamicLib, Path.Combine(OutputDir, destRelative), true);
+
+            if (!StripDebugSymbols)
```

Comment:
this is backwards, it should only do this if debug symbols are stripped otherwise they'll be embedded.
this also shouldn't be done in librarybuilder mode